lib/grape/dsl/validations.rb

```diff
@@ -13,6 +13,9 @@ def reset_validations!
           unset_namespace_stackable :declared_params
           unset_namespace_stackable :validations
           unset_namespace_stackable :params
+          if route_setting(:description)
```

Comment:
But it  only deals with `:description` route setting.
Helpers like `description_field` and `unset_description_field` would be good?

There is also `namespace_setting(:description)` what purpose I cannot realize — it's just not used anywhere. Should this new helpers write to this setting too?